src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -203,16 +203,27 @@ public function cgetAction(Request $request)
         $listBuilder = $this->getListBuilder($request, $fieldDescriptors, $types);
         $listBuilder->setParameter('locale', $locale);
         $listResponse = $listBuilder->execute();
-        $count = $listBuilder->count();
+
+        $previewImageIds = [];
+        foreach ($listResponse as $listItem) {
+            if ($listItem['previewImageId']) {
+                $previewImageIds[] = $listItem['previewImageId'];
+            }
+        }
+        $previewImageFormats = $this->mediaManager->getFormatUrls($previewImageIds, $locale);
 
         for ($i = 0, $length = count($listResponse); $i < $length; ++$i) {
-            $format = $this->formatManager->getFormats(
-                $listResponse[$i]['previewImageId'] ?? $listResponse[$i]['id'],
-                $listResponse[$i]['name'],
-                $listResponse[$i]['version'],
-                $listResponse[$i]['subVersion'],
-                $listResponse[$i]['mimeType']
-            );
+            if ($previewImageId = $listResponse[$i]['previewImageId']) {
```

Comment:
same here before it was fallback to null and nothing did happens if previewImageId was not set